.ci/openshift-ci/cluster/install_kata.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This script installs the built kata-containers in the test cluster,
+# and configure a runtime.
+
+scripts_dir=$(dirname $0)
+deployments_dir=${scripts_dir}/deployments
+configs_dir=${scripts_dir}/configs
+
+source ${scripts_dir}/../../lib.sh
+
+# Wait all worker nodes reboot.
+#
+# Params:
+#   $1 - timeout in seconds.
```

Comment:
Nit (defensive programming): We try to check params at the start of a function:

```bash
wait_for_reboot() {
    local timeout="${1:-}"

    [ -z "$timeout" ] && die "need timeout"

    # ...

}

Reply:
Going to change to `local delta="${1:-1800}"`. So I don't need to pass a repeated value on the two calls made in this script, yet let the function configurable for future usage.